pyproject.toml

```diff
@@ -31,93 +31,88 @@ packages = [{ include = "pymedphys", from = "lib" }]
-watchdog = { version = "*", optional = true }  # groups = ["user", "dev"]
-
-keyring = { version = "*", optional = true }  # groups = ["user", "dev"]
-packaging = { version = "*", optional = true }  # groups = ["user", "dev"]
-PyYAML = { version = "*", optional = true }  # groups = ["user", "dev"]
-requests = { version = "*", optional = true }  # groups = ["user", "dev"]
-python-dateutil = { version = "*", optional = true }  # groups = ["user", "dev", "tests"]
-
-matplotlib = { version = "*", optional = true }  # groups = ["user", "dev"]
-scipy = { version = "*", optional = true }  # groups = ["user", "dev"]
-
-xarray = { version = "*", optional = true }  # groups = ["user", "dev"]
-
-pymssql = { version = "*", optional = true }  # groups = ["user", "dev", "mosaiq"]
-sqlalchemy = { version = "<1.4", optional = true }  # groups = ["user", "dev", "mosaiq"]
-natsort = { version = "*", optional = true }  # groups = ["user", "dev"]
-tomlkit = { version = "*", optional = true }  # groups = ["user", "dev", "propagate"]
-toml = { version = "*", optional = true }  # groups = ["user", "cli", "dev"]
-
-pynetdicom = { version = "*", optional = true }  # groups = ["user", "dev", "dicom"]
-
-dbfread = { version = "*", optional = true }  # groups = ["user", "dev"]
-xmltodict = { version = "*", optional = true }  # groups = ["user", "dev"]
-timeago = { version = "*", optional = true }  # groups = ["user", "dev"]
-xlsxwriter = { version = "*", optional = true }  # groups = ["user", "dev"]
-plotly = { version = "*", optional = true }  # groups = ["user", "dev"]
-fsspec = { version = "*", optional = true }  # groups = ["user", "dev"]
-dicompyler-core = { version = "*", optional = true }  # groups = ["user", "dev"]
-scikit-learn = { version = "*", optional = true }  # groups = ["user", "dev", "mosaiq"]
-
-# Versions of imageio between 2.11 and 2.13 were tested and each got an error
-# similar to the following within the `test_lossless_jpeg` test.
-# $ poetry run pymedphys dev tests -k test_lossless_jpeg
-# imageio.core.request.InitializationError: `TIFF` can not write to `/tmp/tmp6il3r7gi/output.ppm`
-imageio = { version = "<2.11", optional = true }  # groups = ["user", "dev"]
+tqdm = { version = "*", optional = true }     # groups = ["user", "dev", "tests"]
+attrs = { version = "*", optional = true }    # groups = ["user", "dev"]
+watchdog = { version = "*", optional = true } # groups = ["user", "dev"]
+
+keyring = { version = "*", optional = true }         # groups = ["user", "dev"]
+packaging = { version = "*", optional = true }       # groups = ["user", "dev"]
+PyYAML = { version = "*", optional = true }          # groups = ["user", "dev"]
+requests = { version = "*", optional = true }        # groups = ["user", "dev"]
+python-dateutil = { version = "*", optional = true } # groups = ["user", "dev", "tests"]
+
+matplotlib = { version = "*", optional = true } # groups = ["user", "dev"]
+scipy = "1.8.1"                                 # groups = ["user", "dev"]
```

Comment:
I'm assuming we don't want this pin left here? (A left over from debugging?)

Reply:
No sorry - I could've sworn I'd corrected that? Maybe a relic of incorrectly merging auto-changes (propagate?) from Github.